doc/before-after-reboot-checks.md

```diff
@@ -0,0 +1,48 @@
+# Before and After Reboot Checks
+
+CLUO provides optional functionality for running arbitrary actions before and
```

Comment:
CLUO can require custom node annotations before a node is allowed to reboot or after a node has rebooted to allow it to become scheduable.